catalog/spatial/registry/registry-common/src/test/java/org/codice/ddf/registry/common/metacard/TestRegistryObjectMetacardType.java

```diff
@@ -61,7 +48,7 @@ public void testConstruction() {
         assertThat(descriptors, not(nullValue()));
         assertThat(CollectionUtils.isEmpty(descriptors), is(false));
 
-        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.ID)
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(RegistryObjectMetacardType.REGISTRY_ID)
```

Comment:
If you build up that `MetacardType` correctly, he will still have the `Metacard.ID` attribute. Pretty much all metacards should have the core `Metacard` attributes.